egs/wsj/s5/steps/dict/apply_lexicon_edits.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+
+# Copyright 2016  Xiaohui Zhang
+# Apache 2.0.
+
+from __future__ import print_function
+import argparse
+import sys
+
+def GetArgs():
+    parser = argparse.ArgumentParser(description = "Applyccumulate statistics from lattice-alignment"
```

Comment:
applycumulate?
Also there needs to be a comment at the top (and/or a usage message) that explains what this is  about, where its input comes from, gives examples of the format, etc.